frontend/src/components/pages/UserProfilePage.tsx

```diff
@@ -61,6 +61,7 @@ const UserProfilePage = () => {
   const [role, setRole] = useState<string>("");
   const [resumeId, setResumeId] = useState<number | null>(null);
   /* eslint-disable */
+  const [language, setLanguage] = useState<string>("");
```

Comment:
nit: leave a TODO to remove the eslint-disable once the userprofileform is made fully functional